[2024-02-01 00:40:42,661] 	INFO 	27 	emissions_data.py 	<module>() 	Emissions Data Started
[2024-02-01 00:41:59,039] 	INFO 	99 	emissions_data.py 	<module>() 	Emissions Data Ended
[2024-02-01 01:43:27,696] 	INFO 	23 	id_data_v2.py 	<module>() 	ID Data Started
[2024-02-01 01:46:09,639] 	INFO 	120 	id_data_v2.py 	<module>() 	ID Data Ended
[2024-02-01 01:46:10,235] 	INFO 	26 	day_data.py 	<module>() 	Day Data Started
[2024-02-01 01:46:14,075] 	INFO 	72 	day_data.py 	<module>() 	Day Data Ended
[2024-02-01 01:46:14,076] 	INFO 	78 	day_data.py 	<module>() 	Day Data Fusion Started
[2024-02-01 01:46:16,381] 	INFO 	124 	day_data.py 	<module>() 	Day Data Fusion Ended
[2024-02-01 01:46:16,381] 	INFO 	131 	day_data.py 	<module>() 	Day Data Combined Started
[2024-02-01 01:46:20,812] 	INFO 	161 	day_data.py 	<module>() 	Day Data Combined Ended
[2024-02-01 01:46:21,564] 	INFO 	37 	pair_data.py 	<module>() 	Pair Data Started
[2024-02-01 01:47:14,362] 	INFO 	111 	pair_data.py 	<module>() 	Pair Data Ended
[2024-02-01 01:47:14,363] 	INFO 	127 	pair_data.py 	<module>() 	Pair Data Fusion Started
[2024-02-01 01:47:35,607] 	INFO 	194 	pair_data.py 	<module>() 	Pair Data Fusion Ended
[2024-02-01 01:47:35,607] 	INFO 	201 	pair_data.py 	<module>() 	Pair Data Combined Started
[2024-02-01 01:47:58,583] 	INFO 	231 	pair_data.py 	<module>() 	Pair Data Combined Ended
[2024-02-01 01:47:59,386] 	INFO 	38 	bribe_data.py 	<module>() 	Bribe Data Started
[2024-02-01 01:54:44,556] 	ERROR 	148 	bribe_data.py 	<module>() 	Error occurred during Bribe Data process. Error: 'address'
[2024-02-01 01:54:45,353] 	INFO 	38 	fee_data.py 	<module>() 	Fee Data Started
[2024-02-01 01:56:25,183] 	ERROR 	137 	fee_data.py 	<module>() 	Error occurred during Fee Data process. Error: 429 Client Error: Too Many Requests for url: https://rpc.ankr.com/bsc
[2024-02-01 01:56:25,740] 	INFO 	24 	revenue_data.py 	<module>() 	Revenue Data Started
[2024-02-01 01:56:51,930] 	INFO 	72 	revenue_data.py 	<module>() 	Revenue Data Ended
[2024-02-01 01:56:52,447] 	INFO 	24 	revenue_data_v2.py 	<module>() 	Revenue Data Started
[2024-02-01 01:57:11,517] 	INFO 	86 	revenue_data_v2.py 	<module>() 	Revenue Data Ended
[2024-02-01 02:17:30,587] 	INFO 	23 	id_data_v2.py 	<module>() 	ID Data Started
[2024-02-01 02:21:37,433] 	INFO 	120 	id_data_v2.py 	<module>() 	ID Data Ended
[2024-02-01 02:21:38,110] 	INFO 	26 	day_data.py 	<module>() 	Day Data Started
[2024-02-01 02:21:40,797] 	INFO 	72 	day_data.py 	<module>() 	Day Data Ended
[2024-02-01 02:21:40,797] 	INFO 	78 	day_data.py 	<module>() 	Day Data Fusion Started
[2024-02-01 02:21:42,493] 	INFO 	124 	day_data.py 	<module>() 	Day Data Fusion Ended
[2024-02-01 02:21:42,493] 	INFO 	131 	day_data.py 	<module>() 	Day Data Combined Started
[2024-02-01 02:21:45,578] 	INFO 	161 	day_data.py 	<module>() 	Day Data Combined Ended
[2024-02-01 02:21:46,347] 	INFO 	37 	pair_data.py 	<module>() 	Pair Data Started
[2024-02-01 02:22:30,889] 	INFO 	111 	pair_data.py 	<module>() 	Pair Data Ended
[2024-02-01 02:22:30,889] 	INFO 	127 	pair_data.py 	<module>() 	Pair Data Fusion Started
[2024-02-01 02:22:48,577] 	INFO 	194 	pair_data.py 	<module>() 	Pair Data Fusion Ended
[2024-02-01 02:22:48,577] 	INFO 	201 	pair_data.py 	<module>() 	Pair Data Combined Started
[2024-02-01 02:23:05,992] 	INFO 	231 	pair_data.py 	<module>() 	Pair Data Combined Ended
[2024-02-01 02:23:06,807] 	INFO 	38 	bribe_data.py 	<module>() 	Bribe Data Started
[2024-02-01 02:33:16,322] 	ERROR 	148 	bribe_data.py 	<module>() 	Error occurred during Bribe Data process. Error: 'address'
[2024-02-01 02:33:17,125] 	INFO 	38 	fee_data.py 	<module>() 	Fee Data Started
[2024-02-01 02:43:07,321] 	ERROR 	137 	fee_data.py 	<module>() 	Error occurred during Fee Data process. Error: 'address'
[2024-02-01 02:43:07,878] 	INFO 	24 	revenue_data.py 	<module>() 	Revenue Data Started
[2024-02-01 02:43:43,277] 	INFO 	72 	revenue_data.py 	<module>() 	Revenue Data Ended
[2024-02-01 02:43:43,799] 	INFO 	24 	revenue_data_v2.py 	<module>() 	Revenue Data Started
[2024-02-01 02:44:09,882] 	INFO 	86 	revenue_data_v2.py 	<module>() 	Revenue Data Ended
[2024-02-01 03:07:44,109] 	INFO 	23 	id_data_v2.py 	<module>() 	ID Data Started
[2024-02-01 03:10:55,272] 	INFO 	120 	id_data_v2.py 	<module>() 	ID Data Ended
[2024-02-01 03:10:55,836] 	INFO 	26 	day_data.py 	<module>() 	Day Data Started
[2024-02-01 03:10:57,288] 	INFO 	72 	day_data.py 	<module>() 	Day Data Ended
[2024-02-01 03:10:57,289] 	INFO 	78 	day_data.py 	<module>() 	Day Data Fusion Started
[2024-02-01 03:11:11,402] 	ERROR 	126 	day_data.py 	<module>() 	Error occurred during Day Data Fusion process. Error: 'data'
[2024-02-01 03:11:11,402] 	INFO 	131 	day_data.py 	<module>() 	Day Data Combined Started
[2024-02-01 03:11:14,222] 	INFO 	161 	day_data.py 	<module>() 	Day Data Combined Ended
[2024-02-01 03:11:14,981] 	INFO 	37 	pair_data.py 	<module>() 	Pair Data Started
[2024-02-01 03:12:09,719] 	INFO 	111 	pair_data.py 	<module>() 	Pair Data Ended
[2024-02-01 03:12:09,719] 	INFO 	127 	pair_data.py 	<module>() 	Pair Data Fusion Started
[2024-02-01 03:12:25,435] 	ERROR 	159 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aBTCB-WBNB, Address: 0x6B67112aa7b45E8CdC0a93B8D66A6A36e68ae8e5, Error: 'data'
[2024-02-01 03:12:25,555] 	ERROR 	159 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aFTM-WBNB, Address: 0xf6814792050cCD1c603d8a95468d6D9587eCBDAF, Error: 'data'
[2024-02-01 03:12:25,688] 	ERROR 	159 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aWBNB-THE, Address: 0x51Bd5e6d3da9064D59BcaA5A76776560aB42cEb8, Error: 'data'
[2024-02-01 03:12:28,643] 	ERROR 	159 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aANKRBNB-WBNB, Address: 0x2F6C6e00E517944EE5EFE310cd0b98A3fC61Cb98, Error: 'data'
[2024-02-01 03:12:28,805] 	ERROR 	159 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aWBNB-MATIC, Address: 0x775CAc1c1b47250121291934d15912Efb69213E9, Error: 'data'
[2024-02-01 03:12:51,115] 	ERROR 	159 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aHAY-USDT, Address: 0x5B0BAf66718caaBDa49A4Af32eB455c3b99B5821, Error: 'data'
[2024-02-01 03:12:51,311] 	ERROR 	159 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aETH-USDT, Address: 0xa60a504d92a1C95bda729C3F745B361cA822d6dd, Error: 'data'
[2024-02-01 03:12:51,469] 	ERROR 	159 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aWBNB-EDU, Address: 0x4bD39cfb05545c4238316dA69eCadA1b4CaABC05, Error: 'data'
[2024-02-01 03:13:12,068] 	ERROR 	159 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aTWT-WBNB, Address: 0x1A2a83DEaB03dc7110080E7f83DF7FFE36681289, Error: 'data'
[2024-02-01 03:13:12,195] 	ERROR 	159 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aDAO-BUSD, Address: 0x604a99F4c5e46aDD74dBA10C21B5e26374A1162F, Error: 'data'
[2024-02-01 03:13:12,341] 	ERROR 	159 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aMULTI-WBNB, Address: 0x972E8BE53425DbcaF3446c7aC130ADB48ba3E125, Error: 'data'
[2024-02-01 03:13:14,379] 	ERROR 	159 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aHAY-FRXETH, Address: 0xf8A4cDf9eFc4B9b38eaa6e27eE281cB2111fa664, Error: 'data'
[2024-02-01 03:13:18,813] 	ERROR 	159 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aWBNB-OATH, Address: 0x055557c6606f7b0d34e617653c447f079B0B0a73, Error: 'data'
[2024-02-01 03:13:29,938] 	ERROR 	159 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aETH-XCAD, Address: 0xfB0e434EBa0A467cd3f47cEC5De63f4385861eA3, Error: 'data'
[2024-02-01 03:13:30,075] 	ERROR 	159 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aLTC-WBNB, Address: 0x320e85137bB10157BB8BBB22a912D516f8296cb5, Error: 'data'
[2024-02-01 03:13:39,534] 	ERROR 	159 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aFRXETH-FRAX, Address: 0x3CADD2F6A964D262B5Dd5e7169C284B465336F0e, Error: 'data'
[2024-02-01 03:13:39,659] 	ERROR 	159 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aBNBX-USDT, Address: 0x3A8B7E833793Ab906403238853db34BD3231508D, Error: 'data'
[2024-02-01 03:13:41,429] 	ERROR 	159 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aWUSDR-DOLA, Address: 0xdB6BfF81311C5C4F72668D9C4c8a5E2CB4584A82, Error: 'data'
[2024-02-01 03:13:41,550] 	ERROR 	159 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aBNBX-FRXETH, Address: 0x733a0b28e4d7f2CB421730c4E4E26F2aDCe3d240, Error: 'data'
[2024-02-01 03:13:41,675] 	ERROR 	159 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aSTKBNB-THE, Address: 0xF1b7007694DBcba004DB73473CCa3c5aDd3803E4, Error: 'data'
[2024-02-01 03:13:41,795] 	ERROR 	159 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aHAY-WBNB, Address: 0xBF121d987F9635ed6D2f7bB957FBbe163bDEA0E0, Error: 'data'
[2024-02-01 03:13:43,534] 	ERROR 	159 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aBNBX-ETH, Address: 0x6B32E9178a6670dc9b19a74FE5080C54ea5347C7, Error: 'data'
[2024-02-01 03:13:50,223] 	ERROR 	159 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aERN-OATH, Address: 0xd20c7C2693c3Bf844f84DFA03012a6C07032c5a6, Error: 'data'
[2024-02-01 03:13:50,357] 	ERROR 	159 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aBNBX-ERN, Address: 0xc7426Dce489E6b2d580Fd480Ef72D57e5d7643f5, Error: 'data'
[2024-02-01 03:13:50,671] 	ERROR 	159 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aFRAX-ERN, Address: 0x1C2D4F310aeA872C5BD55790D2D78Ca7B31992a3, Error: 'data'
[2024-02-01 03:13:52,127] 	ERROR 	159 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aUSDT-PENDLE, Address: 0x8e3ae967604DC4B4ba676e6D34ac603Fb07fc921, Error: 'data'
[2024-02-01 03:13:53,892] 	ERROR 	159 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aANKRBNB-USDT, Address: 0xc1716bEa36140Fa08476742d80cb28CEcD492A6F, Error: 'data'
[2024-02-01 03:13:54,031] 	ERROR 	159 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aTWT-ANKRBNB, Address: 0x80c264189DD38F4Fa5d6e424C1bf879B3b176076, Error: 'data'
[2024-02-01 03:13:54,464] 	ERROR 	159 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aEQB-WBNB, Address: 0x89217690eb025Ef16124316f4A7884c13b30fb6F, Error: 'data'
[2024-02-01 03:13:56,068] 	ERROR 	159 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aFRXETH-WBNB, Address: 0x6eb8A743723216a0De043c554c85E447B0D59E2e, Error: 'data'
[2024-02-01 03:13:56,841] 	ERROR 	159 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aUSDT-FDUSD, Address: 0x7B879963ae083732f4514d564f4e4613e24E1f67, Error: 'data'
[2024-02-01 03:13:56,968] 	ERROR 	159 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aUSDT-USDV, Address: 0x739e561786C920D025d57eD99be5D4EcA3458E3E, Error: 'data'
[2024-02-01 03:13:57,992] 	ERROR 	159 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aAITECH-WBNB, Address: 0x0485f8cDC8C15BDF180D1Ded90C3ce1275dEaa20, Error: 'data'
[2024-02-01 03:13:58,129] 	ERROR 	159 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aCASH-USDT, Address: 0x45F1Fdb00fB78174196Fa109A27Cdbd07f67F85e, Error: 'data'
[2024-02-01 03:13:58,774] 	ERROR 	159 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aUSDT-TON, Address: 0xD6f0Ba7eC72Ff3974B02c18ED1fC33Da77434d41, Error: 'data'
[2024-02-01 03:13:59,064] 	ERROR 	159 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: IV-3-THE, Address: 0x7c0B5D39765b221810d477E8f02d47a9badF018a, Error: 'data'
[2024-02-01 03:13:59,384] 	ERROR 	159 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aWBNB-ERN, Address: 0xc609daA86FF43F34Eef52C468CbC4934eF8D2502, Error: 'data'
[2024-02-01 03:13:59,679] 	ERROR 	159 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aETH-FRAX, Address: 0x82437aFe50B21d6910498706Dd4Fc7b6567e03eb, Error: 'data'
[2024-02-01 03:14:00,703] 	ERROR 	159 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: IV-9-THE, Address: 0xc7db746EdD16bBdE9a75aa6114400709041dA561, Error: 'data'
[2024-02-01 03:14:00,836] 	ERROR 	159 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aETH-BTCB, Address: 0x1F9B1A3DdeDBf47b96C65F29c0586b678DE2623b, Error: 'data'
[2024-02-01 03:14:06,751] 	INFO 	194 	pair_data.py 	<module>() 	Pair Data Fusion Ended
[2024-02-01 03:14:06,751] 	INFO 	201 	pair_data.py 	<module>() 	Pair Data Combined Started
[2024-02-01 03:14:32,465] 	INFO 	231 	pair_data.py 	<module>() 	Pair Data Combined Ended
[2024-02-01 03:14:33,256] 	INFO 	38 	bribe_data.py 	<module>() 	Bribe Data Started
[2024-02-01 03:22:54,089] 	ERROR 	148 	bribe_data.py 	<module>() 	Error occurred during Bribe Data process. Error: 'address'
[2024-02-01 03:22:54,873] 	INFO 	38 	fee_data.py 	<module>() 	Fee Data Started
[2024-02-01 03:29:58,537] 	ERROR 	137 	fee_data.py 	<module>() 	Error occurred during Fee Data process. Error: 429 Client Error: Too Many Requests for url: https://rpc.ankr.com/bsc
[2024-02-01 03:29:59,082] 	INFO 	24 	revenue_data.py 	<module>() 	Revenue Data Started
[2024-02-01 03:30:08,382] 	INFO 	72 	revenue_data.py 	<module>() 	Revenue Data Ended
[2024-02-01 03:30:08,897] 	INFO 	24 	revenue_data_v2.py 	<module>() 	Revenue Data Started
[2024-02-01 03:30:38,711] 	INFO 	86 	revenue_data_v2.py 	<module>() 	Revenue Data Ended
[2024-02-01 04:09:52,542] 	INFO 	23 	id_data_v2.py 	<module>() 	ID Data Started
[2024-02-01 04:13:25,063] 	INFO 	120 	id_data_v2.py 	<module>() 	ID Data Ended
[2024-02-01 04:13:25,690] 	INFO 	26 	day_data.py 	<module>() 	Day Data Started
[2024-02-01 04:13:29,645] 	INFO 	72 	day_data.py 	<module>() 	Day Data Ended
[2024-02-01 04:13:29,645] 	INFO 	78 	day_data.py 	<module>() 	Day Data Fusion Started
[2024-02-01 04:13:33,259] 	INFO 	124 	day_data.py 	<module>() 	Day Data Fusion Ended
[2024-02-01 04:13:33,259] 	INFO 	131 	day_data.py 	<module>() 	Day Data Combined Started
[2024-02-01 04:13:36,596] 	INFO 	161 	day_data.py 	<module>() 	Day Data Combined Ended
[2024-02-01 04:13:37,366] 	INFO 	37 	pair_data.py 	<module>() 	Pair Data Started
[2024-02-01 04:14:37,698] 	INFO 	111 	pair_data.py 	<module>() 	Pair Data Ended
[2024-02-01 04:14:37,698] 	INFO 	127 	pair_data.py 	<module>() 	Pair Data Fusion Started
[2024-02-01 04:15:01,382] 	INFO 	194 	pair_data.py 	<module>() 	Pair Data Fusion Ended
[2024-02-01 04:15:01,382] 	INFO 	201 	pair_data.py 	<module>() 	Pair Data Combined Started
[2024-02-01 04:15:41,173] 	INFO 	231 	pair_data.py 	<module>() 	Pair Data Combined Ended
[2024-02-01 04:15:41,971] 	INFO 	38 	bribe_data.py 	<module>() 	Bribe Data Started
[2024-02-01 04:24:13,918] 	ERROR 	148 	bribe_data.py 	<module>() 	Error occurred during Bribe Data process. Error: 'address'
[2024-02-01 04:24:14,709] 	INFO 	38 	fee_data.py 	<module>() 	Fee Data Started
[2024-02-01 04:32:36,946] 	ERROR 	137 	fee_data.py 	<module>() 	Error occurred during Fee Data process. Error: 'address'
[2024-02-01 04:32:37,491] 	INFO 	24 	revenue_data.py 	<module>() 	Revenue Data Started
[2024-02-01 04:32:59,032] 	INFO 	72 	revenue_data.py 	<module>() 	Revenue Data Ended
[2024-02-01 04:32:59,551] 	INFO 	24 	revenue_data_v2.py 	<module>() 	Revenue Data Started
[2024-02-01 04:33:22,333] 	INFO 	86 	revenue_data_v2.py 	<module>() 	Revenue Data Ended
[2024-02-01 05:06:31,261] 	INFO 	23 	id_data_v2.py 	<module>() 	ID Data Started
[2024-02-01 05:09:43,710] 	INFO 	120 	id_data_v2.py 	<module>() 	ID Data Ended
[2024-02-01 05:09:44,335] 	INFO 	26 	day_data.py 	<module>() 	Day Data Started
[2024-02-01 05:09:45,898] 	INFO 	72 	day_data.py 	<module>() 	Day Data Ended
[2024-02-01 05:09:45,898] 	INFO 	78 	day_data.py 	<module>() 	Day Data Fusion Started
[2024-02-01 05:09:48,379] 	INFO 	124 	day_data.py 	<module>() 	Day Data Fusion Ended
[2024-02-01 05:09:48,379] 	INFO 	131 	day_data.py 	<module>() 	Day Data Combined Started
[2024-02-01 05:09:50,691] 	INFO 	161 	day_data.py 	<module>() 	Day Data Combined Ended
[2024-02-01 05:09:51,452] 	INFO 	37 	pair_data.py 	<module>() 	Pair Data Started
[2024-02-01 05:10:44,738] 	INFO 	111 	pair_data.py 	<module>() 	Pair Data Ended
[2024-02-01 05:10:44,738] 	INFO 	127 	pair_data.py 	<module>() 	Pair Data Fusion Started
[2024-02-01 05:11:06,276] 	INFO 	194 	pair_data.py 	<module>() 	Pair Data Fusion Ended
[2024-02-01 05:11:06,276] 	INFO 	201 	pair_data.py 	<module>() 	Pair Data Combined Started
[2024-02-01 05:11:26,597] 	INFO 	231 	pair_data.py 	<module>() 	Pair Data Combined Ended
[2024-02-01 05:11:27,403] 	INFO 	38 	bribe_data.py 	<module>() 	Bribe Data Started
[2024-02-01 05:18:56,090] 	ERROR 	148 	bribe_data.py 	<module>() 	Error occurred during Bribe Data process. Error: 'address'
[2024-02-01 05:18:56,888] 	INFO 	38 	fee_data.py 	<module>() 	Fee Data Started
[2024-02-01 05:24:21,436] 	ERROR 	137 	fee_data.py 	<module>() 	Error occurred during Fee Data process. Error: 429 Client Error: Too Many Requests for url: https://rpc.ankr.com/bsc
[2024-02-01 05:24:21,984] 	INFO 	24 	revenue_data.py 	<module>() 	Revenue Data Started
[2024-02-01 05:24:38,530] 	INFO 	72 	revenue_data.py 	<module>() 	Revenue Data Ended
[2024-02-01 05:24:39,054] 	INFO 	24 	revenue_data_v2.py 	<module>() 	Revenue Data Started
[2024-02-01 05:25:05,613] 	INFO 	86 	revenue_data_v2.py 	<module>() 	Revenue Data Ended
[2024-02-01 06:08:52,958] 	INFO 	23 	id_data_v2.py 	<module>() 	ID Data Started
[2024-02-01 06:11:28,000] 	INFO 	120 	id_data_v2.py 	<module>() 	ID Data Ended
[2024-02-01 06:11:28,649] 	INFO 	26 	day_data.py 	<module>() 	Day Data Started
[2024-02-01 06:11:30,765] 	INFO 	72 	day_data.py 	<module>() 	Day Data Ended
[2024-02-01 06:11:30,765] 	INFO 	78 	day_data.py 	<module>() 	Day Data Fusion Started
[2024-02-01 06:11:32,525] 	INFO 	124 	day_data.py 	<module>() 	Day Data Fusion Ended
[2024-02-01 06:11:32,526] 	INFO 	131 	day_data.py 	<module>() 	Day Data Combined Started
[2024-02-01 06:11:35,934] 	INFO 	161 	day_data.py 	<module>() 	Day Data Combined Ended
[2024-02-01 06:11:36,702] 	INFO 	37 	pair_data.py 	<module>() 	Pair Data Started
[2024-02-01 06:12:30,458] 	INFO 	111 	pair_data.py 	<module>() 	Pair Data Ended
[2024-02-01 06:12:30,459] 	INFO 	127 	pair_data.py 	<module>() 	Pair Data Fusion Started
[2024-02-01 06:12:53,702] 	INFO 	194 	pair_data.py 	<module>() 	Pair Data Fusion Ended
[2024-02-01 06:12:53,702] 	INFO 	201 	pair_data.py 	<module>() 	Pair Data Combined Started
[2024-02-01 06:13:21,206] 	INFO 	231 	pair_data.py 	<module>() 	Pair Data Combined Ended
[2024-02-01 06:13:22,010] 	INFO 	38 	bribe_data.py 	<module>() 	Bribe Data Started
[2024-02-01 06:19:34,059] 	ERROR 	148 	bribe_data.py 	<module>() 	Error occurred during Bribe Data process. Error: 429 Client Error: Too Many Requests for url: https://rpc.ankr.com/bsc
[2024-02-01 06:19:34,860] 	INFO 	38 	fee_data.py 	<module>() 	Fee Data Started
[2024-02-01 06:19:35,367] 	ERROR 	137 	fee_data.py 	<module>() 	Error occurred during Fee Data process. Error: 429 Client Error: Too Many Requests for url: https://rpc.ankr.com/bsc
[2024-02-01 06:19:35,912] 	INFO 	24 	revenue_data.py 	<module>() 	Revenue Data Started
[2024-02-01 06:19:46,293] 	INFO 	72 	revenue_data.py 	<module>() 	Revenue Data Ended
[2024-02-01 06:19:46,817] 	INFO 	24 	revenue_data_v2.py 	<module>() 	Revenue Data Started
[2024-02-01 06:19:54,238] 	INFO 	86 	revenue_data_v2.py 	<module>() 	Revenue Data Ended
[2024-02-01 07:07:09,345] 	INFO 	23 	id_data_v2.py 	<module>() 	ID Data Started
[2024-02-01 07:10:27,041] 	INFO 	120 	id_data_v2.py 	<module>() 	ID Data Ended
[2024-02-01 07:10:27,761] 	INFO 	26 	day_data.py 	<module>() 	Day Data Started
[2024-02-01 07:10:29,968] 	INFO 	72 	day_data.py 	<module>() 	Day Data Ended
[2024-02-01 07:10:29,968] 	INFO 	78 	day_data.py 	<module>() 	Day Data Fusion Started
[2024-02-01 07:10:31,602] 	INFO 	124 	day_data.py 	<module>() 	Day Data Fusion Ended
[2024-02-01 07:10:31,602] 	INFO 	131 	day_data.py 	<module>() 	Day Data Combined Started
[2024-02-01 07:12:35,861] 	ERROR 	163 	day_data.py 	<module>() 	Error occurred during Day Data Combined process. Error: {'code': 500, 'message': 'Internal error encountered.', 'status': 'INTERNAL'}
[2024-02-01 07:12:36,693] 	INFO 	37 	pair_data.py 	<module>() 	Pair Data Started
[2024-02-01 07:13:35,788] 	INFO 	111 	pair_data.py 	<module>() 	Pair Data Ended
[2024-02-01 07:13:35,788] 	INFO 	127 	pair_data.py 	<module>() 	Pair Data Fusion Started
[2024-02-01 07:14:03,469] 	INFO 	194 	pair_data.py 	<module>() 	Pair Data Fusion Ended
[2024-02-01 07:14:03,469] 	INFO 	201 	pair_data.py 	<module>() 	Pair Data Combined Started
[2024-02-01 07:14:34,871] 	INFO 	231 	pair_data.py 	<module>() 	Pair Data Combined Ended
[2024-02-01 07:14:35,718] 	INFO 	38 	bribe_data.py 	<module>() 	Bribe Data Started
[2024-02-01 07:22:36,479] 	INFO 	146 	bribe_data.py 	<module>() 	Bribe Data Ended
[2024-02-01 07:22:37,349] 	INFO 	38 	fee_data.py 	<module>() 	Fee Data Started
[2024-02-01 07:28:31,616] 	ERROR 	137 	fee_data.py 	<module>() 	Error occurred during Fee Data process. Error: 429 Client Error: Too Many Requests for url: https://rpc.ankr.com/bsc
[2024-02-01 07:28:32,206] 	INFO 	24 	revenue_data.py 	<module>() 	Revenue Data Started
[2024-02-01 07:28:59,835] 	INFO 	72 	revenue_data.py 	<module>() 	Revenue Data Ended
[2024-02-01 07:29:00,375] 	INFO 	24 	revenue_data_v2.py 	<module>() 	Revenue Data Started
[2024-02-01 07:29:09,170] 	INFO 	86 	revenue_data_v2.py 	<module>() 	Revenue Data Ended
[2024-02-01 08:08:36,190] 	INFO 	23 	id_data_v2.py 	<module>() 	ID Data Started
[2024-02-01 08:11:21,349] 	INFO 	120 	id_data_v2.py 	<module>() 	ID Data Ended
[2024-02-01 08:11:21,974] 	INFO 	26 	day_data.py 	<module>() 	Day Data Started
[2024-02-01 08:11:23,371] 	INFO 	72 	day_data.py 	<module>() 	Day Data Ended
[2024-02-01 08:11:23,371] 	INFO 	78 	day_data.py 	<module>() 	Day Data Fusion Started
[2024-02-01 08:11:24,983] 	INFO 	124 	day_data.py 	<module>() 	Day Data Fusion Ended
[2024-02-01 08:11:24,983] 	INFO 	131 	day_data.py 	<module>() 	Day Data Combined Started
[2024-02-01 08:11:27,202] 	INFO 	161 	day_data.py 	<module>() 	Day Data Combined Ended
[2024-02-01 08:11:27,945] 	INFO 	37 	pair_data.py 	<module>() 	Pair Data Started
[2024-02-01 08:12:16,155] 	INFO 	111 	pair_data.py 	<module>() 	Pair Data Ended
[2024-02-01 08:12:16,156] 	INFO 	127 	pair_data.py 	<module>() 	Pair Data Fusion Started
[2024-02-01 08:12:37,172] 	INFO 	194 	pair_data.py 	<module>() 	Pair Data Fusion Ended
[2024-02-01 08:12:37,172] 	INFO 	201 	pair_data.py 	<module>() 	Pair Data Combined Started
[2024-02-01 08:13:07,971] 	INFO 	231 	pair_data.py 	<module>() 	Pair Data Combined Ended
[2024-02-01 08:13:08,755] 	INFO 	38 	bribe_data.py 	<module>() 	Bribe Data Started
[2024-02-01 08:20:06,046] 	INFO 	146 	bribe_data.py 	<module>() 	Bribe Data Ended
[2024-02-01 08:20:06,835] 	INFO 	38 	fee_data.py 	<module>() 	Fee Data Started
[2024-02-01 08:21:58,022] 	ERROR 	137 	fee_data.py 	<module>() 	Error occurred during Fee Data process. Error: 429 Client Error: Too Many Requests for url: https://rpc.ankr.com/bsc
[2024-02-01 08:21:58,576] 	INFO 	24 	revenue_data.py 	<module>() 	Revenue Data Started
[2024-02-01 08:22:12,055] 	INFO 	72 	revenue_data.py 	<module>() 	Revenue Data Ended
[2024-02-01 08:22:12,573] 	INFO 	24 	revenue_data_v2.py 	<module>() 	Revenue Data Started
[2024-02-01 08:22:20,243] 	INFO 	86 	revenue_data_v2.py 	<module>() 	Revenue Data Ended
[2024-02-01 09:06:33,332] 	INFO 	23 	id_data_v2.py 	<module>() 	ID Data Started
[2024-02-01 09:09:53,272] 	INFO 	120 	id_data_v2.py 	<module>() 	ID Data Ended
[2024-02-01 09:09:53,869] 	INFO 	26 	day_data.py 	<module>() 	Day Data Started
[2024-02-01 09:09:55,777] 	INFO 	72 	day_data.py 	<module>() 	Day Data Ended
[2024-02-01 09:09:55,777] 	INFO 	78 	day_data.py 	<module>() 	Day Data Fusion Started
[2024-02-01 09:09:57,121] 	INFO 	124 	day_data.py 	<module>() 	Day Data Fusion Ended
[2024-02-01 09:09:57,121] 	INFO 	131 	day_data.py 	<module>() 	Day Data Combined Started
[2024-02-01 09:10:00,222] 	INFO 	161 	day_data.py 	<module>() 	Day Data Combined Ended
[2024-02-01 09:10:00,975] 	INFO 	37 	pair_data.py 	<module>() 	Pair Data Started
[2024-02-01 09:10:40,438] 	INFO 	111 	pair_data.py 	<module>() 	Pair Data Ended
[2024-02-01 09:10:40,438] 	INFO 	127 	pair_data.py 	<module>() 	Pair Data Fusion Started
[2024-02-01 09:10:57,283] 	INFO 	194 	pair_data.py 	<module>() 	Pair Data Fusion Ended
[2024-02-01 09:10:57,284] 	INFO 	201 	pair_data.py 	<module>() 	Pair Data Combined Started
[2024-02-01 09:11:30,702] 	INFO 	231 	pair_data.py 	<module>() 	Pair Data Combined Ended
[2024-02-01 09:11:31,493] 	INFO 	38 	bribe_data.py 	<module>() 	Bribe Data Started
[2024-02-01 09:19:47,865] 	INFO 	146 	bribe_data.py 	<module>() 	Bribe Data Ended
[2024-02-01 09:19:48,664] 	INFO 	38 	fee_data.py 	<module>() 	Fee Data Started
[2024-02-01 09:27:54,842] 	ERROR 	137 	fee_data.py 	<module>() 	Error occurred during Fee Data process. Error: 'address'
[2024-02-01 09:27:55,391] 	INFO 	24 	revenue_data.py 	<module>() 	Revenue Data Started
[2024-02-01 09:28:06,542] 	INFO 	72 	revenue_data.py 	<module>() 	Revenue Data Ended
[2024-02-01 09:28:07,060] 	INFO 	24 	revenue_data_v2.py 	<module>() 	Revenue Data Started
[2024-02-01 09:28:14,840] 	INFO 	86 	revenue_data_v2.py 	<module>() 	Revenue Data Ended
[2024-02-01 10:07:14,028] 	INFO 	23 	id_data_v2.py 	<module>() 	ID Data Started
[2024-02-01 10:09:43,390] 	INFO 	120 	id_data_v2.py 	<module>() 	ID Data Ended
[2024-02-01 10:09:43,965] 	INFO 	26 	day_data.py 	<module>() 	Day Data Started
[2024-02-01 10:09:46,146] 	INFO 	72 	day_data.py 	<module>() 	Day Data Ended
[2024-02-01 10:09:46,147] 	INFO 	78 	day_data.py 	<module>() 	Day Data Fusion Started
[2024-02-01 10:09:48,456] 	INFO 	124 	day_data.py 	<module>() 	Day Data Fusion Ended
[2024-02-01 10:09:48,456] 	INFO 	131 	day_data.py 	<module>() 	Day Data Combined Started
[2024-02-01 10:09:51,759] 	INFO 	161 	day_data.py 	<module>() 	Day Data Combined Ended
[2024-02-01 10:09:52,527] 	INFO 	37 	pair_data.py 	<module>() 	Pair Data Started
[2024-02-01 10:10:42,038] 	INFO 	111 	pair_data.py 	<module>() 	Pair Data Ended
[2024-02-01 10:10:42,038] 	INFO 	127 	pair_data.py 	<module>() 	Pair Data Fusion Started
[2024-02-01 10:11:04,732] 	INFO 	194 	pair_data.py 	<module>() 	Pair Data Fusion Ended
[2024-02-01 10:11:04,733] 	INFO 	201 	pair_data.py 	<module>() 	Pair Data Combined Started
[2024-02-01 10:11:34,580] 	INFO 	231 	pair_data.py 	<module>() 	Pair Data Combined Ended
[2024-02-01 10:11:35,398] 	INFO 	38 	bribe_data.py 	<module>() 	Bribe Data Started
[2024-02-01 10:17:40,103] 	INFO 	146 	bribe_data.py 	<module>() 	Bribe Data Ended
[2024-02-01 10:17:40,898] 	INFO 	38 	fee_data.py 	<module>() 	Fee Data Started
[2024-02-01 10:18:31,002] 	ERROR 	137 	fee_data.py 	<module>() 	Error occurred during Fee Data process. Error: 429 Client Error: Too Many Requests for url: https://rpc.ankr.com/bsc
[2024-02-01 10:18:31,597] 	INFO 	24 	revenue_data.py 	<module>() 	Revenue Data Started
[2024-02-01 10:18:45,489] 	INFO 	72 	revenue_data.py 	<module>() 	Revenue Data Ended
[2024-02-01 10:18:46,015] 	INFO 	24 	revenue_data_v2.py 	<module>() 	Revenue Data Started
[2024-02-01 10:18:57,209] 	INFO 	86 	revenue_data_v2.py 	<module>() 	Revenue Data Ended
[2024-02-01 11:04:58,380] 	INFO 	23 	id_data_v2.py 	<module>() 	ID Data Started
[2024-02-01 11:08:02,997] 	INFO 	120 	id_data_v2.py 	<module>() 	ID Data Ended
[2024-02-01 11:08:03,674] 	INFO 	26 	day_data.py 	<module>() 	Day Data Started
[2024-02-01 11:08:05,571] 	INFO 	72 	day_data.py 	<module>() 	Day Data Ended
[2024-02-01 11:08:05,571] 	INFO 	78 	day_data.py 	<module>() 	Day Data Fusion Started
[2024-02-01 11:08:07,480] 	INFO 	124 	day_data.py 	<module>() 	Day Data Fusion Ended
[2024-02-01 11:08:07,480] 	INFO 	131 	day_data.py 	<module>() 	Day Data Combined Started
[2024-02-01 11:11:13,914] 	ERROR 	163 	day_data.py 	<module>() 	Error occurred during Day Data Combined process. Error: {'code': 503, 'message': 'The service is currently unavailable.', 'status': 'UNAVAILABLE'}
[2024-02-01 11:11:14,670] 	INFO 	37 	pair_data.py 	<module>() 	Pair Data Started
[2024-02-01 11:12:12,917] 	INFO 	111 	pair_data.py 	<module>() 	Pair Data Ended
[2024-02-01 11:12:12,917] 	INFO 	127 	pair_data.py 	<module>() 	Pair Data Fusion Started
[2024-02-01 11:12:36,920] 	INFO 	194 	pair_data.py 	<module>() 	Pair Data Fusion Ended
[2024-02-01 11:12:36,920] 	INFO 	201 	pair_data.py 	<module>() 	Pair Data Combined Started
[2024-02-01 11:13:10,894] 	INFO 	231 	pair_data.py 	<module>() 	Pair Data Combined Ended
[2024-02-01 11:13:11,698] 	INFO 	38 	bribe_data.py 	<module>() 	Bribe Data Started
[2024-02-01 11:20:39,924] 	INFO 	146 	bribe_data.py 	<module>() 	Bribe Data Ended
[2024-02-01 11:20:40,721] 	INFO 	38 	fee_data.py 	<module>() 	Fee Data Started
[2024-02-01 11:26:42,446] 	ERROR 	137 	fee_data.py 	<module>() 	Error occurred during Fee Data process. Error: 429 Client Error: Too Many Requests for url: https://rpc.ankr.com/bsc
[2024-02-01 11:26:43,033] 	INFO 	24 	revenue_data.py 	<module>() 	Revenue Data Started
[2024-02-01 11:27:05,080] 	INFO 	72 	revenue_data.py 	<module>() 	Revenue Data Ended
[2024-02-01 11:27:05,616] 	INFO 	24 	revenue_data_v2.py 	<module>() 	Revenue Data Started
[2024-02-01 11:27:18,463] 	INFO 	86 	revenue_data_v2.py 	<module>() 	Revenue Data Ended
[2024-02-01 12:11:09,026] 	INFO 	23 	id_data_v2.py 	<module>() 	ID Data Started
[2024-02-01 12:14:07,941] 	INFO 	120 	id_data_v2.py 	<module>() 	ID Data Ended
[2024-02-01 12:14:08,698] 	INFO 	26 	day_data.py 	<module>() 	Day Data Started
[2024-02-01 12:14:11,658] 	INFO 	72 	day_data.py 	<module>() 	Day Data Ended
[2024-02-01 12:14:11,658] 	INFO 	78 	day_data.py 	<module>() 	Day Data Fusion Started
[2024-02-01 12:14:13,910] 	INFO 	124 	day_data.py 	<module>() 	Day Data Fusion Ended
[2024-02-01 12:14:13,910] 	INFO 	131 	day_data.py 	<module>() 	Day Data Combined Started
[2024-02-01 12:14:16,933] 	INFO 	161 	day_data.py 	<module>() 	Day Data Combined Ended
[2024-02-01 12:14:17,712] 	INFO 	37 	pair_data.py 	<module>() 	Pair Data Started
[2024-02-01 12:15:08,973] 	INFO 	111 	pair_data.py 	<module>() 	Pair Data Ended
[2024-02-01 12:15:08,973] 	INFO 	127 	pair_data.py 	<module>() 	Pair Data Fusion Started
[2024-02-01 12:15:31,637] 	INFO 	194 	pair_data.py 	<module>() 	Pair Data Fusion Ended
[2024-02-01 12:15:31,637] 	INFO 	201 	pair_data.py 	<module>() 	Pair Data Combined Started
[2024-02-01 12:15:51,105] 	INFO 	231 	pair_data.py 	<module>() 	Pair Data Combined Ended
[2024-02-01 12:15:51,929] 	INFO 	38 	bribe_data.py 	<module>() 	Bribe Data Started
[2024-02-01 12:22:53,265] 	INFO 	146 	bribe_data.py 	<module>() 	Bribe Data Ended
[2024-02-01 12:22:54,152] 	INFO 	38 	fee_data.py 	<module>() 	Fee Data Started
[2024-02-01 12:25:36,971] 	ERROR 	137 	fee_data.py 	<module>() 	Error occurred during Fee Data process. Error: 429 Client Error: Too Many Requests for url: https://rpc.ankr.com/bsc
[2024-02-01 12:25:37,595] 	INFO 	24 	revenue_data.py 	<module>() 	Revenue Data Started
[2024-02-01 12:25:51,667] 	INFO 	72 	revenue_data.py 	<module>() 	Revenue Data Ended
[2024-02-01 12:25:52,223] 	INFO 	24 	revenue_data_v2.py 	<module>() 	Revenue Data Started
[2024-02-01 12:26:03,236] 	INFO 	86 	revenue_data_v2.py 	<module>() 	Revenue Data Ended
[2024-02-01 13:06:35,771] 	INFO 	23 	id_data_v2.py 	<module>() 	ID Data Started
[2024-02-01 13:09:02,538] 	INFO 	120 	id_data_v2.py 	<module>() 	ID Data Ended
[2024-02-01 13:09:03,130] 	INFO 	26 	day_data.py 	<module>() 	Day Data Started
[2024-02-01 13:09:05,215] 	INFO 	72 	day_data.py 	<module>() 	Day Data Ended
[2024-02-01 13:09:05,215] 	INFO 	78 	day_data.py 	<module>() 	Day Data Fusion Started
[2024-02-01 13:09:07,989] 	INFO 	124 	day_data.py 	<module>() 	Day Data Fusion Ended
[2024-02-01 13:09:07,989] 	INFO 	131 	day_data.py 	<module>() 	Day Data Combined Started
[2024-02-01 13:09:11,598] 	INFO 	161 	day_data.py 	<module>() 	Day Data Combined Ended
[2024-02-01 13:09:12,421] 	INFO 	37 	pair_data.py 	<module>() 	Pair Data Started
[2024-02-01 13:10:05,413] 	INFO 	111 	pair_data.py 	<module>() 	Pair Data Ended
[2024-02-01 13:10:05,413] 	INFO 	127 	pair_data.py 	<module>() 	Pair Data Fusion Started
[2024-02-01 13:10:37,140] 	INFO 	194 	pair_data.py 	<module>() 	Pair Data Fusion Ended
[2024-02-01 13:10:37,140] 	INFO 	201 	pair_data.py 	<module>() 	Pair Data Combined Started
[2024-02-01 13:11:10,542] 	INFO 	231 	pair_data.py 	<module>() 	Pair Data Combined Ended
[2024-02-01 13:11:11,367] 	INFO 	38 	bribe_data.py 	<module>() 	Bribe Data Started
[2024-02-01 13:17:25,267] 	INFO 	146 	bribe_data.py 	<module>() 	Bribe Data Ended
[2024-02-01 13:17:26,065] 	INFO 	38 	fee_data.py 	<module>() 	Fee Data Started
[2024-02-01 13:18:20,414] 	ERROR 	137 	fee_data.py 	<module>() 	Error occurred during Fee Data process. Error: 429 Client Error: Too Many Requests for url: https://rpc.ankr.com/bsc
[2024-02-01 13:18:21,005] 	INFO 	24 	revenue_data.py 	<module>() 	Revenue Data Started
[2024-02-01 13:18:41,361] 	INFO 	72 	revenue_data.py 	<module>() 	Revenue Data Ended
[2024-02-01 13:18:41,903] 	INFO 	24 	revenue_data_v2.py 	<module>() 	Revenue Data Started
[2024-02-01 13:18:55,006] 	INFO 	86 	revenue_data_v2.py 	<module>() 	Revenue Data Ended
[2024-02-01 14:05:38,059] 	INFO 	23 	id_data_v2.py 	<module>() 	ID Data Started
[2024-02-01 14:09:34,036] 	INFO 	120 	id_data_v2.py 	<module>() 	ID Data Ended
[2024-02-01 14:09:34,743] 	INFO 	26 	day_data.py 	<module>() 	Day Data Started
[2024-02-01 14:09:36,478] 	INFO 	72 	day_data.py 	<module>() 	Day Data Ended
[2024-02-01 14:09:36,478] 	INFO 	78 	day_data.py 	<module>() 	Day Data Fusion Started
[2024-02-01 14:09:39,997] 	INFO 	124 	day_data.py 	<module>() 	Day Data Fusion Ended
[2024-02-01 14:09:39,997] 	INFO 	131 	day_data.py 	<module>() 	Day Data Combined Started
[2024-02-01 14:09:43,116] 	INFO 	161 	day_data.py 	<module>() 	Day Data Combined Ended
[2024-02-01 14:09:43,908] 	INFO 	37 	pair_data.py 	<module>() 	Pair Data Started
[2024-02-01 14:10:28,416] 	INFO 	111 	pair_data.py 	<module>() 	Pair Data Ended
[2024-02-01 14:10:28,416] 	INFO 	127 	pair_data.py 	<module>() 	Pair Data Fusion Started
[2024-02-01 14:10:47,381] 	INFO 	194 	pair_data.py 	<module>() 	Pair Data Fusion Ended
[2024-02-01 14:10:47,381] 	INFO 	201 	pair_data.py 	<module>() 	Pair Data Combined Started
[2024-02-01 14:11:08,311] 	INFO 	231 	pair_data.py 	<module>() 	Pair Data Combined Ended
[2024-02-01 14:11:09,155] 	INFO 	38 	bribe_data.py 	<module>() 	Bribe Data Started
[2024-02-01 14:21:05,304] 	INFO 	146 	bribe_data.py 	<module>() 	Bribe Data Ended
[2024-02-01 14:21:06,205] 	INFO 	38 	fee_data.py 	<module>() 	Fee Data Started
[2024-02-01 14:30:43,805] 	ERROR 	137 	fee_data.py 	<module>() 	Error occurred during Fee Data process. Error: 'address'
[2024-02-01 14:30:44,393] 	INFO 	24 	revenue_data.py 	<module>() 	Revenue Data Started
[2024-02-01 14:31:16,923] 	INFO 	72 	revenue_data.py 	<module>() 	Revenue Data Ended
[2024-02-01 14:31:17,483] 	INFO 	24 	revenue_data_v2.py 	<module>() 	Revenue Data Started
[2024-02-01 14:31:29,309] 	INFO 	86 	revenue_data_v2.py 	<module>() 	Revenue Data Ended
[2024-02-01 15:06:36,391] 	INFO 	23 	id_data_v2.py 	<module>() 	ID Data Started
[2024-02-01 15:10:19,882] 	INFO 	120 	id_data_v2.py 	<module>() 	ID Data Ended
[2024-02-01 15:10:20,655] 	INFO 	26 	day_data.py 	<module>() 	Day Data Started
[2024-02-01 15:10:21,863] 	INFO 	72 	day_data.py 	<module>() 	Day Data Ended
[2024-02-01 15:10:21,863] 	INFO 	78 	day_data.py 	<module>() 	Day Data Fusion Started
[2024-02-01 15:10:23,805] 	INFO 	124 	day_data.py 	<module>() 	Day Data Fusion Ended
[2024-02-01 15:10:23,805] 	INFO 	131 	day_data.py 	<module>() 	Day Data Combined Started
[2024-02-01 15:10:27,091] 	INFO 	161 	day_data.py 	<module>() 	Day Data Combined Ended
[2024-02-01 15:10:27,888] 	INFO 	37 	pair_data.py 	<module>() 	Pair Data Started
[2024-02-01 15:11:21,456] 	INFO 	111 	pair_data.py 	<module>() 	Pair Data Ended
[2024-02-01 15:11:21,456] 	INFO 	127 	pair_data.py 	<module>() 	Pair Data Fusion Started
[2024-02-01 15:11:45,086] 	INFO 	194 	pair_data.py 	<module>() 	Pair Data Fusion Ended
[2024-02-01 15:11:45,086] 	INFO 	201 	pair_data.py 	<module>() 	Pair Data Combined Started
[2024-02-01 15:12:24,317] 	INFO 	231 	pair_data.py 	<module>() 	Pair Data Combined Ended
[2024-02-01 15:12:25,164] 	INFO 	38 	bribe_data.py 	<module>() 	Bribe Data Started
[2024-02-01 15:20:50,117] 	INFO 	146 	bribe_data.py 	<module>() 	Bribe Data Ended
[2024-02-01 15:20:50,956] 	INFO 	38 	fee_data.py 	<module>() 	Fee Data Started
[2024-02-01 15:29:08,917] 	ERROR 	137 	fee_data.py 	<module>() 	Error occurred during Fee Data process. Error: 'address'
[2024-02-01 15:29:09,495] 	INFO 	24 	revenue_data.py 	<module>() 	Revenue Data Started
[2024-02-01 15:29:39,244] 	INFO 	72 	revenue_data.py 	<module>() 	Revenue Data Ended
[2024-02-01 15:29:39,767] 	INFO 	24 	revenue_data_v2.py 	<module>() 	Revenue Data Started
[2024-02-01 15:29:52,970] 	INFO 	86 	revenue_data_v2.py 	<module>() 	Revenue Data Ended
[2024-02-01 16:08:07,033] 	INFO 	23 	id_data_v2.py 	<module>() 	ID Data Started
[2024-02-01 16:12:09,172] 	INFO 	120 	id_data_v2.py 	<module>() 	ID Data Ended
[2024-02-01 16:12:09,959] 	INFO 	26 	day_data.py 	<module>() 	Day Data Started
[2024-02-01 16:12:12,036] 	INFO 	72 	day_data.py 	<module>() 	Day Data Ended
[2024-02-01 16:12:12,036] 	INFO 	78 	day_data.py 	<module>() 	Day Data Fusion Started
[2024-02-01 16:12:14,891] 	INFO 	124 	day_data.py 	<module>() 	Day Data Fusion Ended
[2024-02-01 16:12:14,892] 	INFO 	131 	day_data.py 	<module>() 	Day Data Combined Started
[2024-02-01 16:12:17,238] 	INFO 	161 	day_data.py 	<module>() 	Day Data Combined Ended
[2024-02-01 16:12:18,068] 	INFO 	37 	pair_data.py 	<module>() 	Pair Data Started
[2024-02-01 16:13:02,640] 	INFO 	111 	pair_data.py 	<module>() 	Pair Data Ended
[2024-02-01 16:13:02,640] 	INFO 	127 	pair_data.py 	<module>() 	Pair Data Fusion Started
[2024-02-01 16:13:22,973] 	INFO 	194 	pair_data.py 	<module>() 	Pair Data Fusion Ended
[2024-02-01 16:13:22,973] 	INFO 	201 	pair_data.py 	<module>() 	Pair Data Combined Started
[2024-02-01 16:13:55,341] 	INFO 	231 	pair_data.py 	<module>() 	Pair Data Combined Ended
[2024-02-01 16:13:56,204] 	INFO 	38 	bribe_data.py 	<module>() 	Bribe Data Started
[2024-02-01 16:24:49,666] 	INFO 	146 	bribe_data.py 	<module>() 	Bribe Data Ended
[2024-02-01 16:24:50,550] 	INFO 	38 	fee_data.py 	<module>() 	Fee Data Started
[2024-02-01 16:34:49,848] 	ERROR 	137 	fee_data.py 	<module>() 	Error occurred during Fee Data process. Error: 'address'
[2024-02-01 16:34:50,444] 	INFO 	24 	revenue_data.py 	<module>() 	Revenue Data Started
[2024-02-01 16:35:09,976] 	INFO 	72 	revenue_data.py 	<module>() 	Revenue Data Ended
[2024-02-01 16:35:10,564] 	INFO 	24 	revenue_data_v2.py 	<module>() 	Revenue Data Started
[2024-02-01 16:35:24,247] 	INFO 	86 	revenue_data_v2.py 	<module>() 	Revenue Data Ended
[2024-02-01 17:06:24,229] 	INFO 	23 	id_data_v2.py 	<module>() 	ID Data Started
[2024-02-01 17:10:47,498] 	INFO 	120 	id_data_v2.py 	<module>() 	ID Data Ended
[2024-02-01 17:10:48,263] 	INFO 	26 	day_data.py 	<module>() 	Day Data Started
[2024-02-01 17:10:50,226] 	INFO 	72 	day_data.py 	<module>() 	Day Data Ended
[2024-02-01 17:10:50,226] 	INFO 	78 	day_data.py 	<module>() 	Day Data Fusion Started
[2024-02-01 17:10:51,570] 	INFO 	124 	day_data.py 	<module>() 	Day Data Fusion Ended
[2024-02-01 17:10:51,570] 	INFO 	131 	day_data.py 	<module>() 	Day Data Combined Started
[2024-02-01 17:10:55,555] 	INFO 	161 	day_data.py 	<module>() 	Day Data Combined Ended
[2024-02-01 17:10:56,312] 	INFO 	37 	pair_data.py 	<module>() 	Pair Data Started
[2024-02-01 17:11:40,273] 	INFO 	111 	pair_data.py 	<module>() 	Pair Data Ended
[2024-02-01 17:11:40,273] 	INFO 	127 	pair_data.py 	<module>() 	Pair Data Fusion Started
[2024-02-01 17:12:00,555] 	INFO 	194 	pair_data.py 	<module>() 	Pair Data Fusion Ended
[2024-02-01 17:12:00,555] 	INFO 	201 	pair_data.py 	<module>() 	Pair Data Combined Started
[2024-02-01 17:12:28,289] 	INFO 	231 	pair_data.py 	<module>() 	Pair Data Combined Ended
[2024-02-01 17:12:29,089] 	INFO 	38 	bribe_data.py 	<module>() 	Bribe Data Started
[2024-02-01 17:23:48,147] 	INFO 	146 	bribe_data.py 	<module>() 	Bribe Data Ended
[2024-02-01 17:23:48,959] 	INFO 	38 	fee_data.py 	<module>() 	Fee Data Started
[2024-02-01 17:34:42,955] 	ERROR 	137 	fee_data.py 	<module>() 	Error occurred during Fee Data process. Error: 'address'
[2024-02-01 17:34:43,522] 	INFO 	24 	revenue_data.py 	<module>() 	Revenue Data Started
[2024-02-01 17:35:09,412] 	INFO 	72 	revenue_data.py 	<module>() 	Revenue Data Ended
[2024-02-01 17:35:09,933] 	INFO 	24 	revenue_data_v2.py 	<module>() 	Revenue Data Started
[2024-02-01 17:35:21,623] 	INFO 	86 	revenue_data_v2.py 	<module>() 	Revenue Data Ended
[2024-02-01 18:08:06,996] 	INFO 	23 	id_data_v2.py 	<module>() 	ID Data Started
[2024-02-01 18:12:48,657] 	INFO 	120 	id_data_v2.py 	<module>() 	ID Data Ended
[2024-02-01 18:12:49,361] 	INFO 	26 	day_data.py 	<module>() 	Day Data Started
[2024-02-01 18:12:52,402] 	INFO 	72 	day_data.py 	<module>() 	Day Data Ended
[2024-02-01 18:12:52,402] 	INFO 	78 	day_data.py 	<module>() 	Day Data Fusion Started
[2024-02-01 18:12:55,155] 	INFO 	124 	day_data.py 	<module>() 	Day Data Fusion Ended
[2024-02-01 18:12:55,155] 	INFO 	131 	day_data.py 	<module>() 	Day Data Combined Started
[2024-02-01 18:12:59,159] 	INFO 	161 	day_data.py 	<module>() 	Day Data Combined Ended
[2024-02-01 18:12:59,917] 	INFO 	37 	pair_data.py 	<module>() 	Pair Data Started
[2024-02-01 18:13:46,735] 	INFO 	111 	pair_data.py 	<module>() 	Pair Data Ended
[2024-02-01 18:13:46,735] 	INFO 	127 	pair_data.py 	<module>() 	Pair Data Fusion Started
[2024-02-01 18:14:07,678] 	INFO 	194 	pair_data.py 	<module>() 	Pair Data Fusion Ended
[2024-02-01 18:14:07,678] 	INFO 	201 	pair_data.py 	<module>() 	Pair Data Combined Started
[2024-02-01 18:14:44,668] 	INFO 	231 	pair_data.py 	<module>() 	Pair Data Combined Ended
[2024-02-01 18:14:45,473] 	INFO 	38 	bribe_data.py 	<module>() 	Bribe Data Started
[2024-02-01 18:25:55,386] 	INFO 	146 	bribe_data.py 	<module>() 	Bribe Data Ended
[2024-02-01 18:25:56,186] 	INFO 	38 	fee_data.py 	<module>() 	Fee Data Started
[2024-02-01 18:36:41,846] 	ERROR 	137 	fee_data.py 	<module>() 	Error occurred during Fee Data process. Error: 'address'
[2024-02-01 18:36:42,413] 	INFO 	24 	revenue_data.py 	<module>() 	Revenue Data Started
[2024-02-01 18:37:04,072] 	INFO 	72 	revenue_data.py 	<module>() 	Revenue Data Ended
[2024-02-01 18:37:04,590] 	INFO 	24 	revenue_data_v2.py 	<module>() 	Revenue Data Started
[2024-02-01 18:37:18,608] 	INFO 	86 	revenue_data_v2.py 	<module>() 	Revenue Data Ended
[2024-02-01 19:06:20,823] 	INFO 	23 	id_data_v2.py 	<module>() 	ID Data Started
[2024-02-01 19:09:45,745] 	INFO 	120 	id_data_v2.py 	<module>() 	ID Data Ended
[2024-02-01 19:09:46,363] 	INFO 	26 	day_data.py 	<module>() 	Day Data Started
[2024-02-01 19:09:47,908] 	INFO 	72 	day_data.py 	<module>() 	Day Data Ended
[2024-02-01 19:09:47,908] 	INFO 	78 	day_data.py 	<module>() 	Day Data Fusion Started
[2024-02-01 19:09:49,944] 	INFO 	124 	day_data.py 	<module>() 	Day Data Fusion Ended
[2024-02-01 19:09:49,944] 	INFO 	131 	day_data.py 	<module>() 	Day Data Combined Started
[2024-02-01 19:09:52,622] 	INFO 	161 	day_data.py 	<module>() 	Day Data Combined Ended
[2024-02-01 19:09:53,397] 	INFO 	37 	pair_data.py 	<module>() 	Pair Data Started
[2024-02-01 19:10:52,836] 	INFO 	111 	pair_data.py 	<module>() 	Pair Data Ended
[2024-02-01 19:10:52,836] 	INFO 	127 	pair_data.py 	<module>() 	Pair Data Fusion Started
[2024-02-01 19:11:16,675] 	INFO 	194 	pair_data.py 	<module>() 	Pair Data Fusion Ended
[2024-02-01 19:11:16,675] 	INFO 	201 	pair_data.py 	<module>() 	Pair Data Combined Started
[2024-02-01 19:11:36,128] 	INFO 	231 	pair_data.py 	<module>() 	Pair Data Combined Ended
[2024-02-01 19:11:36,943] 	INFO 	38 	bribe_data.py 	<module>() 	Bribe Data Started
[2024-02-01 19:20:29,726] 	INFO 	146 	bribe_data.py 	<module>() 	Bribe Data Ended
[2024-02-01 19:20:30,565] 	INFO 	38 	fee_data.py 	<module>() 	Fee Data Started
[2024-02-01 19:29:17,395] 	ERROR 	137 	fee_data.py 	<module>() 	Error occurred during Fee Data process. Error: 'address'
[2024-02-01 19:29:17,952] 	INFO 	24 	revenue_data.py 	<module>() 	Revenue Data Started
[2024-02-01 19:29:32,914] 	INFO 	72 	revenue_data.py 	<module>() 	Revenue Data Ended
[2024-02-01 19:29:33,441] 	INFO 	24 	revenue_data_v2.py 	<module>() 	Revenue Data Started
[2024-02-01 19:29:45,174] 	INFO 	86 	revenue_data_v2.py 	<module>() 	Revenue Data Ended
[2024-02-01 20:06:41,589] 	INFO 	23 	id_data_v2.py 	<module>() 	ID Data Started
[2024-02-01 20:10:23,306] 	INFO 	120 	id_data_v2.py 	<module>() 	ID Data Ended
[2024-02-01 20:10:23,952] 	INFO 	26 	day_data.py 	<module>() 	Day Data Started
[2024-02-01 20:10:25,371] 	INFO 	72 	day_data.py 	<module>() 	Day Data Ended
[2024-02-01 20:10:25,371] 	INFO 	78 	day_data.py 	<module>() 	Day Data Fusion Started
[2024-02-01 20:10:27,272] 	INFO 	124 	day_data.py 	<module>() 	Day Data Fusion Ended
[2024-02-01 20:10:27,273] 	INFO 	131 	day_data.py 	<module>() 	Day Data Combined Started
[2024-02-01 20:10:29,807] 	INFO 	161 	day_data.py 	<module>() 	Day Data Combined Ended
[2024-02-01 20:10:30,597] 	INFO 	37 	pair_data.py 	<module>() 	Pair Data Started
[2024-02-01 20:11:26,609] 	INFO 	111 	pair_data.py 	<module>() 	Pair Data Ended
[2024-02-01 20:11:26,609] 	INFO 	127 	pair_data.py 	<module>() 	Pair Data Fusion Started
[2024-02-01 20:11:49,568] 	INFO 	194 	pair_data.py 	<module>() 	Pair Data Fusion Ended
[2024-02-01 20:11:49,569] 	INFO 	201 	pair_data.py 	<module>() 	Pair Data Combined Started
[2024-02-01 20:12:24,976] 	INFO 	231 	pair_data.py 	<module>() 	Pair Data Combined Ended
[2024-02-01 20:12:25,814] 	INFO 	38 	bribe_data.py 	<module>() 	Bribe Data Started
[2024-02-01 20:21:13,845] 	INFO 	146 	bribe_data.py 	<module>() 	Bribe Data Ended
[2024-02-01 20:21:14,729] 	INFO 	38 	fee_data.py 	<module>() 	Fee Data Started
[2024-02-01 20:29:38,238] 	ERROR 	137 	fee_data.py 	<module>() 	Error occurred during Fee Data process. Error: 'address'
[2024-02-01 20:29:38,833] 	INFO 	24 	revenue_data.py 	<module>() 	Revenue Data Started
[2024-02-01 20:29:51,165] 	INFO 	72 	revenue_data.py 	<module>() 	Revenue Data Ended
[2024-02-01 20:29:51,716] 	INFO 	24 	revenue_data_v2.py 	<module>() 	Revenue Data Started
[2024-02-01 20:30:01,989] 	INFO 	86 	revenue_data_v2.py 	<module>() 	Revenue Data Ended
[2024-02-01 21:04:24,420] 	INFO 	23 	id_data_v2.py 	<module>() 	ID Data Started
[2024-02-01 21:07:05,105] 	INFO 	120 	id_data_v2.py 	<module>() 	ID Data Ended
[2024-02-01 21:07:05,834] 	INFO 	26 	day_data.py 	<module>() 	Day Data Started
[2024-02-01 21:07:07,975] 	INFO 	72 	day_data.py 	<module>() 	Day Data Ended
[2024-02-01 21:07:07,975] 	INFO 	78 	day_data.py 	<module>() 	Day Data Fusion Started
[2024-02-01 21:07:10,957] 	INFO 	124 	day_data.py 	<module>() 	Day Data Fusion Ended
[2024-02-01 21:07:10,957] 	INFO 	131 	day_data.py 	<module>() 	Day Data Combined Started
[2024-02-01 21:07:14,870] 	INFO 	161 	day_data.py 	<module>() 	Day Data Combined Ended
[2024-02-01 21:07:15,627] 	INFO 	37 	pair_data.py 	<module>() 	Pair Data Started
[2024-02-01 21:08:09,609] 	INFO 	111 	pair_data.py 	<module>() 	Pair Data Ended
[2024-02-01 21:08:09,610] 	INFO 	127 	pair_data.py 	<module>() 	Pair Data Fusion Started
[2024-02-01 21:08:33,313] 	INFO 	194 	pair_data.py 	<module>() 	Pair Data Fusion Ended
[2024-02-01 21:08:33,314] 	INFO 	201 	pair_data.py 	<module>() 	Pair Data Combined Started
[2024-02-01 21:08:55,398] 	INFO 	231 	pair_data.py 	<module>() 	Pair Data Combined Ended
[2024-02-01 21:08:56,216] 	INFO 	38 	bribe_data.py 	<module>() 	Bribe Data Started
[2024-02-01 21:15:20,890] 	INFO 	146 	bribe_data.py 	<module>() 	Bribe Data Ended
[2024-02-01 21:15:21,708] 	INFO 	38 	fee_data.py 	<module>() 	Fee Data Started
[2024-02-01 21:16:33,393] 	ERROR 	137 	fee_data.py 	<module>() 	Error occurred during Fee Data process. Error: 429 Client Error: Too Many Requests for url: https://rpc.ankr.com/bsc
[2024-02-01 21:16:33,954] 	INFO 	24 	revenue_data.py 	<module>() 	Revenue Data Started
[2024-02-01 21:16:44,960] 	INFO 	72 	revenue_data.py 	<module>() 	Revenue Data Ended
[2024-02-01 21:16:45,543] 	INFO 	24 	revenue_data_v2.py 	<module>() 	Revenue Data Started
[2024-02-01 21:16:58,517] 	ERROR 	88 	revenue_data_v2.py 	<module>() 	Error occurred during Revenue Data process. Error: HTTP Error 502: Bad Gateway
[2024-02-01 22:06:53,311] 	INFO 	23 	id_data_v2.py 	<module>() 	ID Data Started
[2024-02-01 22:10:24,848] 	INFO 	120 	id_data_v2.py 	<module>() 	ID Data Ended
[2024-02-01 22:10:25,604] 	INFO 	26 	day_data.py 	<module>() 	Day Data Started
[2024-02-01 22:10:28,701] 	INFO 	72 	day_data.py 	<module>() 	Day Data Ended
[2024-02-01 22:10:28,701] 	INFO 	78 	day_data.py 	<module>() 	Day Data Fusion Started
[2024-02-01 22:10:30,712] 	INFO 	124 	day_data.py 	<module>() 	Day Data Fusion Ended
[2024-02-01 22:10:30,712] 	INFO 	131 	day_data.py 	<module>() 	Day Data Combined Started
[2024-02-01 22:10:34,720] 	INFO 	161 	day_data.py 	<module>() 	Day Data Combined Ended
[2024-02-01 22:10:35,488] 	INFO 	37 	pair_data.py 	<module>() 	Pair Data Started
[2024-02-01 22:11:36,698] 	INFO 	111 	pair_data.py 	<module>() 	Pair Data Ended
[2024-02-01 22:11:36,698] 	INFO 	127 	pair_data.py 	<module>() 	Pair Data Fusion Started
[2024-02-01 22:12:01,519] 	INFO 	194 	pair_data.py 	<module>() 	Pair Data Fusion Ended
[2024-02-01 22:12:01,520] 	INFO 	201 	pair_data.py 	<module>() 	Pair Data Combined Started
[2024-02-01 22:12:31,125] 	INFO 	231 	pair_data.py 	<module>() 	Pair Data Combined Ended
[2024-02-01 22:12:31,921] 	INFO 	38 	bribe_data.py 	<module>() 	Bribe Data Started
[2024-02-01 22:21:27,403] 	INFO 	146 	bribe_data.py 	<module>() 	Bribe Data Ended
[2024-02-01 22:21:28,198] 	INFO 	38 	fee_data.py 	<module>() 	Fee Data Started
[2024-02-01 22:29:43,260] 	ERROR 	137 	fee_data.py 	<module>() 	Error occurred during Fee Data process. Error: Could not decode contract function call to rewardsListLength with return data: b'', output_types: ['uint256']
[2024-02-01 22:29:43,804] 	INFO 	24 	revenue_data.py 	<module>() 	Revenue Data Started
[2024-02-01 22:30:03,791] 	INFO 	72 	revenue_data.py 	<module>() 	Revenue Data Ended
[2024-02-01 22:30:04,309] 	INFO 	24 	revenue_data_v2.py 	<module>() 	Revenue Data Started
[2024-02-01 22:30:18,460] 	INFO 	86 	revenue_data_v2.py 	<module>() 	Revenue Data Ended
[2024-02-01 23:06:32,095] 	INFO 	23 	id_data_v2.py 	<module>() 	ID Data Started
[2024-02-01 23:10:12,930] 	INFO 	120 	id_data_v2.py 	<module>() 	ID Data Ended
[2024-02-01 23:10:13,524] 	INFO 	26 	day_data.py 	<module>() 	Day Data Started
[2024-02-01 23:10:14,888] 	INFO 	72 	day_data.py 	<module>() 	Day Data Ended
[2024-02-01 23:10:14,888] 	INFO 	78 	day_data.py 	<module>() 	Day Data Fusion Started
[2024-02-01 23:10:16,740] 	INFO 	124 	day_data.py 	<module>() 	Day Data Fusion Ended
[2024-02-01 23:10:16,740] 	INFO 	131 	day_data.py 	<module>() 	Day Data Combined Started
[2024-02-01 23:10:20,205] 	INFO 	161 	day_data.py 	<module>() 	Day Data Combined Ended
[2024-02-01 23:10:20,963] 	INFO 	37 	pair_data.py 	<module>() 	Pair Data Started
[2024-02-01 23:11:16,275] 	INFO 	111 	pair_data.py 	<module>() 	Pair Data Ended
[2024-02-01 23:11:16,275] 	INFO 	127 	pair_data.py 	<module>() 	Pair Data Fusion Started
[2024-02-01 23:11:39,355] 	INFO 	194 	pair_data.py 	<module>() 	Pair Data Fusion Ended
[2024-02-01 23:11:39,355] 	INFO 	201 	pair_data.py 	<module>() 	Pair Data Combined Started
[2024-02-01 23:12:01,860] 	INFO 	231 	pair_data.py 	<module>() 	Pair Data Combined Ended
[2024-02-01 23:12:02,656] 	INFO 	38 	bribe_data.py 	<module>() 	Bribe Data Started
[2024-02-01 23:20:34,441] 	INFO 	146 	bribe_data.py 	<module>() 	Bribe Data Ended
[2024-02-01 23:20:35,262] 	INFO 	38 	fee_data.py 	<module>() 	Fee Data Started
[2024-02-01 23:28:31,148] 	ERROR 	137 	fee_data.py 	<module>() 	Error occurred during Fee Data process. Error: 'address'
[2024-02-01 23:28:31,693] 	INFO 	24 	revenue_data.py 	<module>() 	Revenue Data Started
[2024-02-01 23:28:51,413] 	INFO 	72 	revenue_data.py 	<module>() 	Revenue Data Ended
[2024-02-01 23:28:51,929] 	INFO 	24 	revenue_data_v2.py 	<module>() 	Revenue Data Started
[2024-02-01 23:29:03,614] 	INFO 	86 	revenue_data_v2.py 	<module>() 	Revenue Data Ended
[2024-02-02 00:30:22,186] 	INFO 	23 	id_data_v2.py 	<module>() 	ID Data Started
[2024-02-02 00:34:46,977] 	INFO 	120 	id_data_v2.py 	<module>() 	ID Data Ended
[2024-02-02 00:34:47,638] 	INFO 	26 	day_data.py 	<module>() 	Day Data Started
[2024-02-02 00:34:49,742] 	INFO 	72 	day_data.py 	<module>() 	Day Data Ended
[2024-02-02 00:34:49,742] 	INFO 	78 	day_data.py 	<module>() 	Day Data Fusion Started
[2024-02-02 00:34:50,953] 	INFO 	124 	day_data.py 	<module>() 	Day Data Fusion Ended
[2024-02-02 00:34:50,953] 	INFO 	131 	day_data.py 	<module>() 	Day Data Combined Started
[2024-02-02 00:34:54,936] 	INFO 	161 	day_data.py 	<module>() 	Day Data Combined Ended
[2024-02-02 00:34:55,692] 	INFO 	37 	pair_data.py 	<module>() 	Pair Data Started
[2024-02-02 00:35:36,440] 	INFO 	111 	pair_data.py 	<module>() 	Pair Data Ended
[2024-02-02 00:35:36,440] 	INFO 	127 	pair_data.py 	<module>() 	Pair Data Fusion Started
[2024-02-02 00:35:54,121] 	INFO 	194 	pair_data.py 	<module>() 	Pair Data Fusion Ended
[2024-02-02 00:35:54,122] 	INFO 	201 	pair_data.py 	<module>() 	Pair Data Combined Started
[2024-02-02 00:36:45,092] 	INFO 	231 	pair_data.py 	<module>() 	Pair Data Combined Ended
[2024-02-02 00:36:45,890] 	INFO 	38 	bribe_data.py 	<module>() 	Bribe Data Started
[2024-02-02 00:47:16,111] 	INFO 	146 	bribe_data.py 	<module>() 	Bribe Data Ended
[2024-02-02 00:47:16,909] 	INFO 	38 	fee_data.py 	<module>() 	Fee Data Started
[2024-02-02 00:57:49,627] 	ERROR 	137 	fee_data.py 	<module>() 	Error occurred during Fee Data process. Error: 'address'
[2024-02-02 00:57:50,184] 	INFO 	24 	revenue_data.py 	<module>() 	Revenue Data Started
[2024-02-02 00:58:08,541] 	INFO 	72 	revenue_data.py 	<module>() 	Revenue Data Ended
[2024-02-02 00:58:09,058] 	INFO 	24 	revenue_data_v2.py 	<module>() 	Revenue Data Started
[2024-02-02 00:58:25,678] 	INFO 	86 	revenue_data_v2.py 	<module>() 	Revenue Data Ended
[2024-02-02 01:39:24,703] 	INFO 	23 	id_data_v2.py 	<module>() 	ID Data Started
[2024-02-02 01:43:06,640] 	INFO 	120 	id_data_v2.py 	<module>() 	ID Data Ended
[2024-02-02 01:43:07,451] 	INFO 	26 	day_data.py 	<module>() 	Day Data Started
[2024-02-02 01:43:11,510] 	INFO 	72 	day_data.py 	<module>() 	Day Data Ended
[2024-02-02 01:43:11,510] 	INFO 	78 	day_data.py 	<module>() 	Day Data Fusion Started
[2024-02-02 01:43:15,346] 	INFO 	124 	day_data.py 	<module>() 	Day Data Fusion Ended
[2024-02-02 01:43:15,346] 	INFO 	131 	day_data.py 	<module>() 	Day Data Combined Started
[2024-02-02 01:43:19,708] 	INFO 	161 	day_data.py 	<module>() 	Day Data Combined Ended
[2024-02-02 01:43:20,469] 	INFO 	37 	pair_data.py 	<module>() 	Pair Data Started
[2024-02-02 01:44:16,582] 	INFO 	111 	pair_data.py 	<module>() 	Pair Data Ended
[2024-02-02 01:44:16,582] 	INFO 	127 	pair_data.py 	<module>() 	Pair Data Fusion Started
[2024-02-02 01:44:41,693] 	INFO 	194 	pair_data.py 	<module>() 	Pair Data Fusion Ended
[2024-02-02 01:44:41,693] 	INFO 	201 	pair_data.py 	<module>() 	Pair Data Combined Started
[2024-02-02 01:45:12,828] 	INFO 	231 	pair_data.py 	<module>() 	Pair Data Combined Ended
[2024-02-02 01:45:13,626] 	INFO 	38 	bribe_data.py 	<module>() 	Bribe Data Started
[2024-02-02 01:53:57,565] 	INFO 	146 	bribe_data.py 	<module>() 	Bribe Data Ended
[2024-02-02 01:53:58,351] 	INFO 	38 	fee_data.py 	<module>() 	Fee Data Started
[2024-02-02 02:02:47,020] 	ERROR 	137 	fee_data.py 	<module>() 	Error occurred during Fee Data process. Error: 'address'
[2024-02-02 02:02:47,563] 	INFO 	24 	revenue_data.py 	<module>() 	Revenue Data Started
[2024-02-02 02:03:06,903] 	INFO 	72 	revenue_data.py 	<module>() 	Revenue Data Ended
[2024-02-02 02:03:07,422] 	INFO 	24 	revenue_data_v2.py 	<module>() 	Revenue Data Started
[2024-02-02 02:03:22,383] 	INFO 	86 	revenue_data_v2.py 	<module>() 	Revenue Data Ended
[2024-02-02 02:13:06,868] 	INFO 	23 	id_data_v2.py 	<module>() 	ID Data Started
[2024-02-02 02:16:46,305] 	INFO 	120 	id_data_v2.py 	<module>() 	ID Data Ended
[2024-02-02 02:16:46,873] 	INFO 	26 	day_data.py 	<module>() 	Day Data Started
[2024-02-02 02:16:49,148] 	INFO 	72 	day_data.py 	<module>() 	Day Data Ended
[2024-02-02 02:16:49,148] 	INFO 	78 	day_data.py 	<module>() 	Day Data Fusion Started
[2024-02-02 02:16:52,144] 	INFO 	124 	day_data.py 	<module>() 	Day Data Fusion Ended
[2024-02-02 02:16:52,145] 	INFO 	131 	day_data.py 	<module>() 	Day Data Combined Started
[2024-02-02 02:16:55,258] 	INFO 	161 	day_data.py 	<module>() 	Day Data Combined Ended
[2024-02-02 02:16:56,008] 	INFO 	37 	pair_data.py 	<module>() 	Pair Data Started
[2024-02-02 02:17:49,901] 	INFO 	111 	pair_data.py 	<module>() 	Pair Data Ended
[2024-02-02 02:17:49,901] 	INFO 	127 	pair_data.py 	<module>() 	Pair Data Fusion Started
[2024-02-02 02:18:19,779] 	INFO 	194 	pair_data.py 	<module>() 	Pair Data Fusion Ended
[2024-02-02 02:18:19,779] 	INFO 	201 	pair_data.py 	<module>() 	Pair Data Combined Started
[2024-02-02 02:18:53,605] 	INFO 	231 	pair_data.py 	<module>() 	Pair Data Combined Ended
[2024-02-02 02:18:54,413] 	INFO 	38 	bribe_data.py 	<module>() 	Bribe Data Started
[2024-02-02 02:27:40,376] 	INFO 	146 	bribe_data.py 	<module>() 	Bribe Data Ended
[2024-02-02 02:27:41,175] 	INFO 	38 	fee_data.py 	<module>() 	Fee Data Started
[2024-02-02 02:36:16,327] 	ERROR 	137 	fee_data.py 	<module>() 	Error occurred during Fee Data process. Error: 'address'
[2024-02-02 02:36:16,878] 	INFO 	24 	revenue_data.py 	<module>() 	Revenue Data Started
[2024-02-02 02:36:34,010] 	INFO 	72 	revenue_data.py 	<module>() 	Revenue Data Ended
[2024-02-02 02:36:34,534] 	INFO 	24 	revenue_data_v2.py 	<module>() 	Revenue Data Started
[2024-02-02 02:36:51,967] 	INFO 	86 	revenue_data_v2.py 	<module>() 	Revenue Data Ended
[2024-02-02 03:07:36,349] 	INFO 	23 	id_data_v2.py 	<module>() 	ID Data Started
[2024-02-02 03:11:44,092] 	INFO 	120 	id_data_v2.py 	<module>() 	ID Data Ended
[2024-02-02 03:11:44,704] 	INFO 	26 	day_data.py 	<module>() 	Day Data Started
[2024-02-02 03:11:47,871] 	INFO 	72 	day_data.py 	<module>() 	Day Data Ended
[2024-02-02 03:11:47,871] 	INFO 	78 	day_data.py 	<module>() 	Day Data Fusion Started
[2024-02-02 03:11:50,863] 	INFO 	124 	day_data.py 	<module>() 	Day Data Fusion Ended
[2024-02-02 03:11:50,863] 	INFO 	131 	day_data.py 	<module>() 	Day Data Combined Started
[2024-02-02 03:11:54,553] 	INFO 	161 	day_data.py 	<module>() 	Day Data Combined Ended
[2024-02-02 03:11:55,312] 	INFO 	37 	pair_data.py 	<module>() 	Pair Data Started
[2024-02-02 03:12:38,778] 	INFO 	111 	pair_data.py 	<module>() 	Pair Data Ended
[2024-02-02 03:12:38,779] 	INFO 	127 	pair_data.py 	<module>() 	Pair Data Fusion Started
[2024-02-02 03:12:56,172] 	INFO 	194 	pair_data.py 	<module>() 	Pair Data Fusion Ended
[2024-02-02 03:12:56,172] 	INFO 	201 	pair_data.py 	<module>() 	Pair Data Combined Started
[2024-02-02 03:13:14,415] 	INFO 	231 	pair_data.py 	<module>() 	Pair Data Combined Ended
[2024-02-02 03:13:15,221] 	INFO 	38 	bribe_data.py 	<module>() 	Bribe Data Started
[2024-02-02 03:23:40,404] 	INFO 	146 	bribe_data.py 	<module>() 	Bribe Data Ended
[2024-02-02 03:23:41,214] 	INFO 	38 	fee_data.py 	<module>() 	Fee Data Started
[2024-02-02 03:33:28,019] 	ERROR 	137 	fee_data.py 	<module>() 	Error occurred during Fee Data process. Error: 'address'
[2024-02-02 03:33:28,579] 	INFO 	24 	revenue_data.py 	<module>() 	Revenue Data Started
[2024-02-02 03:33:42,985] 	INFO 	72 	revenue_data.py 	<module>() 	Revenue Data Ended
[2024-02-02 03:33:43,518] 	INFO 	24 	revenue_data_v2.py 	<module>() 	Revenue Data Started
[2024-02-02 03:33:55,067] 	INFO 	86 	revenue_data_v2.py 	<module>() 	Revenue Data Ended
[2024-02-02 04:08:11,763] 	INFO 	23 	id_data_v2.py 	<module>() 	ID Data Started
[2024-02-02 04:11:22,857] 	INFO 	120 	id_data_v2.py 	<module>() 	ID Data Ended
[2024-02-02 04:11:23,518] 	INFO 	26 	day_data.py 	<module>() 	Day Data Started
[2024-02-02 04:11:24,981] 	INFO 	72 	day_data.py 	<module>() 	Day Data Ended
[2024-02-02 04:11:24,981] 	INFO 	78 	day_data.py 	<module>() 	Day Data Fusion Started
[2024-02-02 04:11:26,820] 	INFO 	124 	day_data.py 	<module>() 	Day Data Fusion Ended
[2024-02-02 04:11:26,820] 	INFO 	131 	day_data.py 	<module>() 	Day Data Combined Started
[2024-02-02 04:11:29,218] 	INFO 	161 	day_data.py 	<module>() 	Day Data Combined Ended
[2024-02-02 04:11:29,989] 	INFO 	37 	pair_data.py 	<module>() 	Pair Data Started
[2024-02-02 04:12:19,833] 	INFO 	111 	pair_data.py 	<module>() 	Pair Data Ended
[2024-02-02 04:12:19,833] 	INFO 	127 	pair_data.py 	<module>() 	Pair Data Fusion Started
[2024-02-02 04:12:39,977] 	INFO 	194 	pair_data.py 	<module>() 	Pair Data Fusion Ended
[2024-02-02 04:12:39,977] 	INFO 	201 	pair_data.py 	<module>() 	Pair Data Combined Started
[2024-02-02 04:13:01,410] 	INFO 	231 	pair_data.py 	<module>() 	Pair Data Combined Ended
[2024-02-02 04:13:02,223] 	INFO 	38 	bribe_data.py 	<module>() 	Bribe Data Started
[2024-02-02 04:20:44,928] 	INFO 	146 	bribe_data.py 	<module>() 	Bribe Data Ended
[2024-02-02 04:20:45,757] 	INFO 	38 	fee_data.py 	<module>() 	Fee Data Started
[2024-02-02 04:27:37,051] 	ERROR 	137 	fee_data.py 	<module>() 	Error occurred during Fee Data process. Error: 429 Client Error: Too Many Requests for url: https://rpc.ankr.com/bsc
[2024-02-02 04:27:37,626] 	INFO 	24 	revenue_data.py 	<module>() 	Revenue Data Started
[2024-02-02 04:27:57,514] 	INFO 	72 	revenue_data.py 	<module>() 	Revenue Data Ended
[2024-02-02 04:27:58,056] 	INFO 	24 	revenue_data_v2.py 	<module>() 	Revenue Data Started
[2024-02-02 04:28:14,489] 	INFO 	86 	revenue_data_v2.py 	<module>() 	Revenue Data Ended
[2024-02-02 05:06:42,522] 	INFO 	23 	id_data_v2.py 	<module>() 	ID Data Started
[2024-02-02 05:09:46,657] 	INFO 	120 	id_data_v2.py 	<module>() 	ID Data Ended
[2024-02-02 05:09:47,490] 	INFO 	26 	day_data.py 	<module>() 	Day Data Started
[2024-02-02 05:09:49,328] 	INFO 	72 	day_data.py 	<module>() 	Day Data Ended
[2024-02-02 05:09:49,328] 	INFO 	78 	day_data.py 	<module>() 	Day Data Fusion Started
[2024-02-02 05:09:50,949] 	INFO 	124 	day_data.py 	<module>() 	Day Data Fusion Ended
[2024-02-02 05:09:50,949] 	INFO 	131 	day_data.py 	<module>() 	Day Data Combined Started
[2024-02-02 05:09:53,319] 	INFO 	161 	day_data.py 	<module>() 	Day Data Combined Ended
[2024-02-02 05:09:54,074] 	INFO 	37 	pair_data.py 	<module>() 	Pair Data Started
[2024-02-02 05:10:44,686] 	INFO 	111 	pair_data.py 	<module>() 	Pair Data Ended
[2024-02-02 05:10:44,687] 	INFO 	127 	pair_data.py 	<module>() 	Pair Data Fusion Started
[2024-02-02 05:11:06,665] 	INFO 	194 	pair_data.py 	<module>() 	Pair Data Fusion Ended
[2024-02-02 05:11:06,665] 	INFO 	201 	pair_data.py 	<module>() 	Pair Data Combined Started
[2024-02-02 05:11:30,288] 	INFO 	231 	pair_data.py 	<module>() 	Pair Data Combined Ended
[2024-02-02 05:11:31,090] 	INFO 	38 	bribe_data.py 	<module>() 	Bribe Data Started
[2024-02-02 05:19:24,527] 	INFO 	146 	bribe_data.py 	<module>() 	Bribe Data Ended
[2024-02-02 05:19:25,330] 	INFO 	38 	fee_data.py 	<module>() 	Fee Data Started
[2024-02-02 05:24:10,931] 	ERROR 	137 	fee_data.py 	<module>() 	Error occurred during Fee Data process. Error: 429 Client Error: Too Many Requests for url: https://rpc.ankr.com/bsc
[2024-02-02 05:24:11,487] 	INFO 	24 	revenue_data.py 	<module>() 	Revenue Data Started
[2024-02-02 05:24:39,772] 	INFO 	72 	revenue_data.py 	<module>() 	Revenue Data Ended
[2024-02-02 05:24:40,294] 	INFO 	24 	revenue_data_v2.py 	<module>() 	Revenue Data Started
[2024-02-02 05:24:58,910] 	INFO 	86 	revenue_data_v2.py 	<module>() 	Revenue Data Ended
[2024-02-02 06:08:44,369] 	INFO 	23 	id_data_v2.py 	<module>() 	ID Data Started
[2024-02-02 06:11:33,292] 	INFO 	120 	id_data_v2.py 	<module>() 	ID Data Ended
[2024-02-02 06:11:34,032] 	INFO 	26 	day_data.py 	<module>() 	Day Data Started
[2024-02-02 06:11:37,189] 	INFO 	72 	day_data.py 	<module>() 	Day Data Ended
[2024-02-02 06:11:37,189] 	INFO 	78 	day_data.py 	<module>() 	Day Data Fusion Started
[2024-02-02 06:11:38,744] 	INFO 	124 	day_data.py 	<module>() 	Day Data Fusion Ended
[2024-02-02 06:11:38,744] 	INFO 	131 	day_data.py 	<module>() 	Day Data Combined Started
[2024-02-02 06:11:42,694] 	INFO 	161 	day_data.py 	<module>() 	Day Data Combined Ended
[2024-02-02 06:11:43,491] 	INFO 	37 	pair_data.py 	<module>() 	Pair Data Started
[2024-02-02 06:12:38,276] 	INFO 	111 	pair_data.py 	<module>() 	Pair Data Ended
[2024-02-02 06:12:38,276] 	INFO 	127 	pair_data.py 	<module>() 	Pair Data Fusion Started
[2024-02-02 06:13:00,013] 	INFO 	194 	pair_data.py 	<module>() 	Pair Data Fusion Ended
[2024-02-02 06:13:00,013] 	INFO 	201 	pair_data.py 	<module>() 	Pair Data Combined Started
[2024-02-02 06:13:26,455] 	INFO 	231 	pair_data.py 	<module>() 	Pair Data Combined Ended
[2024-02-02 06:13:27,252] 	INFO 	38 	bribe_data.py 	<module>() 	Bribe Data Started
[2024-02-02 06:19:51,187] 	ERROR 	148 	bribe_data.py 	<module>() 	Error occurred during Bribe Data process. Error: 429 Client Error: Too Many Requests for url: https://rpc.ankr.com/bsc
[2024-02-02 06:19:51,999] 	INFO 	38 	fee_data.py 	<module>() 	Fee Data Started
[2024-02-02 06:19:52,446] 	ERROR 	137 	fee_data.py 	<module>() 	Error occurred during Fee Data process. Error: 429 Client Error: Too Many Requests for url: https://rpc.ankr.com/bsc
[2024-02-02 06:19:52,989] 	INFO 	24 	revenue_data.py 	<module>() 	Revenue Data Started
[2024-02-02 06:20:09,013] 	INFO 	72 	revenue_data.py 	<module>() 	Revenue Data Ended
[2024-02-02 06:20:09,537] 	INFO 	24 	revenue_data_v2.py 	<module>() 	Revenue Data Started
[2024-02-02 06:20:29,495] 	INFO 	86 	revenue_data_v2.py 	<module>() 	Revenue Data Ended
[2024-02-02 07:05:52,220] 	INFO 	23 	id_data_v2.py 	<module>() 	ID Data Started
[2024-02-02 07:08:38,886] 	INFO 	120 	id_data_v2.py 	<module>() 	ID Data Ended
[2024-02-02 07:08:39,619] 	INFO 	26 	day_data.py 	<module>() 	Day Data Started
[2024-02-02 07:08:41,695] 	INFO 	72 	day_data.py 	<module>() 	Day Data Ended
[2024-02-02 07:08:41,696] 	INFO 	78 	day_data.py 	<module>() 	Day Data Fusion Started
[2024-02-02 07:08:43,110] 	INFO 	124 	day_data.py 	<module>() 	Day Data Fusion Ended
[2024-02-02 07:08:43,110] 	INFO 	131 	day_data.py 	<module>() 	Day Data Combined Started
[2024-02-02 07:08:46,457] 	INFO 	161 	day_data.py 	<module>() 	Day Data Combined Ended
[2024-02-02 07:08:47,230] 	INFO 	37 	pair_data.py 	<module>() 	Pair Data Started
[2024-02-02 07:09:43,082] 	INFO 	111 	pair_data.py 	<module>() 	Pair Data Ended
[2024-02-02 07:09:43,082] 	INFO 	127 	pair_data.py 	<module>() 	Pair Data Fusion Started
[2024-02-02 07:10:06,550] 	INFO 	194 	pair_data.py 	<module>() 	Pair Data Fusion Ended
[2024-02-02 07:10:06,550] 	INFO 	201 	pair_data.py 	<module>() 	Pair Data Combined Started
[2024-02-02 07:10:48,528] 	INFO 	231 	pair_data.py 	<module>() 	Pair Data Combined Ended
[2024-02-02 07:10:49,354] 	INFO 	38 	bribe_data.py 	<module>() 	Bribe Data Started
[2024-02-02 07:17:30,052] 	INFO 	146 	bribe_data.py 	<module>() 	Bribe Data Ended
[2024-02-02 07:17:30,881] 	INFO 	38 	fee_data.py 	<module>() 	Fee Data Started
[2024-02-02 07:18:29,520] 	ERROR 	137 	fee_data.py 	<module>() 	Error occurred during Fee Data process. Error: 429 Client Error: Too Many Requests for url: https://rpc.ankr.com/bsc
[2024-02-02 07:18:30,119] 	INFO 	24 	revenue_data.py 	<module>() 	Revenue Data Started
[2024-02-02 07:18:48,790] 	INFO 	72 	revenue_data.py 	<module>() 	Revenue Data Ended
[2024-02-02 07:18:49,340] 	INFO 	24 	revenue_data_v2.py 	<module>() 	Revenue Data Started
[2024-02-02 07:19:11,187] 	INFO 	86 	revenue_data_v2.py 	<module>() 	Revenue Data Ended
[2024-02-02 08:08:43,911] 	INFO 	23 	id_data_v2.py 	<module>() 	ID Data Started
[2024-02-02 08:12:19,258] 	INFO 	120 	id_data_v2.py 	<module>() 	ID Data Ended
[2024-02-02 08:12:19,870] 	INFO 	26 	day_data.py 	<module>() 	Day Data Started
[2024-02-02 08:12:23,166] 	INFO 	72 	day_data.py 	<module>() 	Day Data Ended
[2024-02-02 08:12:23,166] 	INFO 	78 	day_data.py 	<module>() 	Day Data Fusion Started
[2024-02-02 08:12:26,020] 	INFO 	124 	day_data.py 	<module>() 	Day Data Fusion Ended
[2024-02-02 08:12:26,020] 	INFO 	131 	day_data.py 	<module>() 	Day Data Combined Started
[2024-02-02 08:12:32,200] 	INFO 	161 	day_data.py 	<module>() 	Day Data Combined Ended
[2024-02-02 08:12:32,955] 	INFO 	37 	pair_data.py 	<module>() 	Pair Data Started
[2024-02-02 08:13:11,225] 	INFO 	111 	pair_data.py 	<module>() 	Pair Data Ended
[2024-02-02 08:13:11,225] 	INFO 	127 	pair_data.py 	<module>() 	Pair Data Fusion Started
[2024-02-02 08:13:28,790] 	INFO 	194 	pair_data.py 	<module>() 	Pair Data Fusion Ended
[2024-02-02 08:13:28,790] 	INFO 	201 	pair_data.py 	<module>() 	Pair Data Combined Started
[2024-02-02 08:14:02,059] 	INFO 	231 	pair_data.py 	<module>() 	Pair Data Combined Ended
[2024-02-02 08:14:02,861] 	INFO 	38 	bribe_data.py 	<module>() 	Bribe Data Started
[2024-02-02 08:23:06,908] 	INFO 	146 	bribe_data.py 	<module>() 	Bribe Data Ended
[2024-02-02 08:23:07,709] 	INFO 	38 	fee_data.py 	<module>() 	Fee Data Started
[2024-02-02 08:31:23,408] 	ERROR 	137 	fee_data.py 	<module>() 	Error occurred during Fee Data process. Error: 'address'
[2024-02-02 08:31:23,960] 	INFO 	24 	revenue_data.py 	<module>() 	Revenue Data Started
[2024-02-02 08:31:36,163] 	INFO 	72 	revenue_data.py 	<module>() 	Revenue Data Ended
[2024-02-02 08:31:36,677] 	INFO 	24 	revenue_data_v2.py 	<module>() 	Revenue Data Started
[2024-02-02 08:31:55,068] 	INFO 	86 	revenue_data_v2.py 	<module>() 	Revenue Data Ended
